### PR TITLE
Add chinese translation

### PR DESCRIPTION
**Pull request by zuoyuanchun**
Add chinese translation for CloseAllFrameworkAssembliesCommand

### CODE REVIEW

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx [hunk 1]
```diff
@@ -1324,6 +1324,15 @@
   <data name="DbgSettings_EnableManagedDebuggingAssistants" xml:space="preserve">
     <value>启用托管调试助手 (MDA)</value>
   </data>
+  <data name="DbgSettings_EnableJustMyCodeDebugging" xml:space="preserve">
+    <value>启用 "调试代码过滤"</value>
```
Comment (UlyssesWu) [inline Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx:1328]: 请使用VS中的表达方式，“调试代码过滤”并没有更好地表达这个意思。
image

Comment (zuoyuanchun) [reply Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx]: 非常感谢,已提交修改

**Review by UlyssesWu — APPROVED** [review]

**Review by UlyssesWu — APPROVED** [review]

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx [hunk 2]
```diff
@@ -1225,6 +1254,18 @@
   <data name="StartDebuggingCommand" xml:space="preserve">
     <value>开始调试</value>
   </data>
+  <data name="StartDebuggingWarning_UnknownBitness" xml:space="preserve">
+    <value>无法确定目标文件的位数。调试可能会导致不需要的或者不受控制的图像执行。
```
Comment (UlyssesWu) [inline Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx:1258]: 由于image（映像）一词引发误解颇多，如果有替代说法，尽量使用替代说法。

> 无法确定目标文件的位数。调试可能会导致**非预期的**或者不受控制的**程序映像**执行。

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx [hunk 3]
```diff
@@ -1225,6 +1254,18 @@
...
+
+是否继续？</value>
+  </data>
+  <data name="StartDebuggingWarning_IncorrectBitness" xml:space="preserve">
+    <value>目标文件的位数与当前版本的 dnSpy 不匹配。调试可能会导致不需要的或者不受控制的图像执行。
+
+建议使用 32 位 dnSpy 调试 32 位图像，使用 64 位 dnSpy 调试 64 位图像。
```
Comment (UlyssesWu) [inline Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx:1265]: 建议使用 32 位 dnSpy 调试 32 位**程序**，使用 64 位 dnSpy 调试 64 位**程序**。

**Review by UlyssesWu — CHANGES_REQUESTED** [review]
No obvious problem for other CN translations.

> 1. Deleted spaces in some resource files.

👎 Personally, I disagree with it. These spaces are always there when Visual Studio creates the resx file. There's no need to remove them, unless a code style rule is set by maintainers.

> 2. Replaced "\n" with "\r\n" in some resource files (replaced OA with 0D 0A).

👍 OK.

> 3. Added some untranslated strings (Chinese and some Russian).
> 4. To facilitate comparison of missing translation entries, I adjusted the code order (with the default language resource file as a reference).
> 5. Some Russian strings are relatively long, and I am worried that they will exceed the display part of the control, so I did not translate them. I hope friends who know Russian can help translate them.
> 6. My non-native language level is very poor, so I use Google for comparison and translation. Of course, for some common words in some software, I refer to them as references.

👎 Why not just waiting for a native Russian speaker to translate them? Especially when your PR name is "Add chinese translation". If you insist to add Russian translations, I'd suggest you to create another PR.

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx (hunk 2 above)
Comment (zuoyuanchun) [reply Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx]: 已修改,非常感谢您的建议

File: Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx (hunk 3 above)
Comment (zuoyuanchun) [reply Extensions/dnSpy.Debugger/dnSpy.Debugger/Properties/dnSpy.Debugger.Resources.zh-CN.resx]: 已修改,非常感谢您的建议

**Review by UlyssesWu — APPROVED** [review]

**Pull request merged**